### PR TITLE
Separate existing styles for WaitForTransaction to a base and a theme

### PR DESCRIPTION
**Pull request by JustCaptcha**

### CODE REVIEW

File: universal-login-react/src/ui/commons/WaitingForTransaction.tsx [hunk 1]
```diff
@@ -16,19 +20,21 @@ const Body = ({action, relayerConfig, transactionHash, className, info}: Waiting
       action={action}
       className={className}
     />
-    <div className="modal-pending-section">
-      <h3 className="transaction-hash-title">Transaction hash</h3>
+    <div className={`${classForComponent('waitingfor-modal-pending-section')}`}>
+      <h3 className={`${classForComponent('waitingfortransaction-transaction-hash-title')}`}>Transaction hash</h3>
```
Comment (JustynaBroniszewska) [inline universal-login-react/src/ui/commons/WaitingForTransaction.tsx]:
```
<h3 className={classForComponent('waitingfortransaction-transaction-hash-title')}>Transaction hash</h3>
```

File: universal-login-react/src/ui/styles/base/waitingFor.sass [hunk 2]
```diff
@@ -0,0 +1,22 @@
+.unilogin-component-waitingfor
```
Comment (JustynaBroniszewska) [inline universal-login-react/src/ui/styles/base/waitingFor.sass]: We can remove it

File: universal-login-react/src/ui/styles/base/waitingForTransaction.sass [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+.unilogin-component-waitingfortransaction-transaction-hash-title
+  text-align: left
+
+
+.unilogin-component-waitingfortransaction-info-text
+  position: relative
+  padding-left: 33px
+
+
+
```
Comment (JustynaBroniszewska) [inline universal-login-react/src/ui/styles/base/waitingForTransaction.sass]: Too many empty lines. One is enough

File: universal-login-react/src/ui/styles/themes/Legacy/waitingForTransactionThemeLegacy.sass [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+  border: 1px solid #E1E1F0
+
+.unilogin-theme-default .unilogin-component-waitingfortransaction-txn-hash-link
+  font-size: 13px
+  text-overflow: ellipsis
+  color: #fff
+
+.unilogin-theme-default .unilogin-component-waitingfortransaction-modal-pending-section
+
+
```
Comment (JustynaBroniszewska) [inline universal-login-react/src/ui/styles/themes/Legacy/waitingForTransactionThemeLegacy.sass]: We can remove it

File: universal-login-react/src/ui/commons/ExplorerLink.tsx [hunk 5]
```diff
@@ -1,17 +1,18 @@
...
 import {getEtherscanUrl} from '../../core/utils/getEtherscanUrl';
+import '../styles/base/waitingForTransaction.sass';
 
 interface ExplorerLinkProps {
   chainName: string;
   transactionHash?: string;
 }
 
 export const ExplorerLink = ({chainName, transactionHash}: ExplorerLinkProps) => (
-  <p className="txn-hash-text">
```
Comment (xitronix) [inline universal-login-react/src/ui/commons/ExplorerLink.tsx]: #NextPr classForComponent

File: universal-login-react/src/ui/styles/themes/Legacy/waitingForTransactionThemeLegacy.sass [hunk 6]
```diff
@@ -0,0 +1,23 @@
...
+.unilogin-theme-default .unilogin-component-waitingfortransaction-transaction-hash-title
+  font-size: 14px
+  color: #B9C3F
+
+
```
Comment (xitronix) [inline universal-login-react/src/ui/styles/themes/Legacy/waitingForTransactionThemeLegacy.sass]: should be 1 new line

File: universal-login-react/src/ui/themes/Theme.tsx [hunk 7]
```diff
@@ -6,10 +6,11 @@ export const ThemeContext = createContext<ContextProps>(['default', (newTheme: s
 
 export type ThemeProps = {
   children: ReactNode;
+  initialTheme?: string;
```
Comment (xitronix) [inline universal-login-react/src/ui/themes/Theme.tsx]: just theme or themeName

File: universal-login-react/src/ui/commons/WaitingFor.tsx [hunk 8]
```diff
@@ -11,14 +10,14 @@ export interface WaitingForProps {
 
 export const WaitingFor = ({action, className}: WaitingForProps) => {
   return (
-    <div className="universal-login-waiting-for">
+    <div className="universal-login-waitingfor">
       <div className={getStyleForTopLevelComponent(className)}>
```
Comment (xitronix) [inline universal-login-react/src/ui/commons/WaitingFor.tsx]: Should we change it for themeForComponent

File: universal-login-react/src/ui/commons/WaitingForTransaction.tsx [hunk 9]
```diff
@@ -16,19 +20,21 @@ const Body = ({action, relayerConfig, transactionHash, className, info}: Waiting
...
-    {info && <p className="info-text">{info}</p>}
+    <div className={'unilogin-component'}>
+      {info && <p className={classForComponent("info-text")}>{info}</p>}
+    </div>
   </div>
 );
 
 export const WaitingForTransaction = (props: WaitingForTransactionProps) => (
-  <div className="universal-login-waiting-for-transaction">
+  <div className={`unilogin-component-waitingfortransaction ${useThemeClassFor()}`}>
```
Comment (JustynaBroniszewska) [inline universal-login-react/src/ui/commons/WaitingForTransaction.tsx]: #NextPR we can use here helper function `classForComponent()`

File: universal-login-react/src/ui/commons/WaitingForTransaction.tsx [hunk 10]
```diff
@@ -16,19 +20,21 @@ const Body = ({action, relayerConfig, transactionHash, className, info}: Waiting
...
-      <h3 className="transaction-hash-title">Transaction hash</h3>
+    <div className={classForComponent('waitingfor-modal-pending-section')}>
+      <h3 className={classForComponent('waitingfortransaction-transaction-hash-title')}>Transaction hash</h3>
       <ExplorerLink
         chainName={relayerConfig.chainSpec.name}
         transactionHash={transactionHash}
       />
     </div>
-    {info && <p className="info-text">{info}</p>}
+    <div className={'unilogin-component'}>
```
Comment (JustynaBroniszewska) [inline universal-login-react/src/ui/commons/WaitingForTransaction.tsx]: It can be removed now

**Pull request merged**